### PR TITLE
Set the master version of DC/OS to 2.1.0-dev.

### PR DESCRIPTION
**Pull request by orsenthil**
## High-level description

* DCOS-58854 - Set the master version of DC/OS to 2.1.0-dev.

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Comment by orsenthil** [comment]
Merged this directly when the tests were successful. Post commit reviews are welcome.

File: CHANGES.md [hunk 1]
```diff
@@ -1,112 +1,23 @@
...
-* --cli-telemetry-disabled
-* --validate-config
-* --preflight
-* --install-prereqs
-* --deploy
-* --postflight
-
-The Marathon option `MARATHON_DEFAULT_ACCEPTED_RESOURCE_ROLES` has been deprecated. Please use `MARATHON_ACCEPTED_RESOURCE_ROLES_DEFAULT_BEHAVIOR`, instead, which has valid values of `any`, `unreserved`, or `reserved`.
 
 ### Fixed and improved
```
Comment (jgehrcke) [inline CHANGES.md]: Hey Senthil, this section should not be here.

It was accidentally added by this PR:
https://github.com/dcos/dcos/commit/47bcb77eb91fc707e9b0b3c120875dde3a2db747#r35098787

Comment (orsenthil) [reply CHANGES.md]: Thanks for the review, JP. I have addressed it here: https://github.com/dcos/dcos/pull/6264

**Pull request merged**